### PR TITLE
feat: Tolgee Translator small text and visual updates

### PR DESCRIPTION
**Pull request by Martin005**

### CODE REVIEW

File: blog/2023-06-06-tolgee-translator.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ tags: [translation, ai, chatgpt, in-context, sdk, i18n]
 
 Do you think that AI will replace human translators? AI has made its way into various aspects of our lives, including localization, where it demonstrates exceptional capabilities. Yet, two significant challenges hinder the complete adoption of AI for software translation: reliability and context. Tolgee AI Translator aims to tackle these challenges head-on to provide contextually tailored translations of superior quality.
 
-![Tolgee translator](/img/blog/tolgee-translator/post-ai.png)
+![Tolgee AI translator](/img/blog/tolgee-translator/post-ai.png)
```
Comment (JanCizmar) [inline blog/2023-06-06-tolgee-translator.md:12]: Have we agreed on this? Personally I would not flex it so much. All the translators use some kind of AI, so one may ask "What about it?"

Comment (JanCizmar) [reply blog/2023-06-06-tolgee-translator.md:12]: @stepan662

Comment (Martin005) [reply blog/2023-06-06-tolgee-translator.md:12]: @JanCizmar I was just making the naming consistent.

Comment (stepan662) [reply blog/2023-06-06-tolgee-translator.md:12]: I also feel that everything is AI these days, it's kinda cheap label

**Review by JanCizmar — APPROVED** [review]

**Pull request merged**